src/lib/cli_lib/commands.ml

```diff
@@ -110,8 +111,18 @@ let validate_transaction =
                   Rosetta_lib.Transaction.to_mina_signed transaction_json
                 with
                 | Ok cmd ->
-                    if Mina_base.Signed_command.check_signature cmd then
-                      Format.eprintf "Transaction was valid@."
+                    if
+                      Mina_base.Signed_command.check_signature
+                        ~signature_kind:Mainnet cmd
+                    then
+                      Format.eprintf
```

Comment:
why stderr for the output?

Reply:
I think the intention was for only the final line to be on stdout, since the rest is auxiliary logging info. Should all of the messages be on stdout?

Reply:
The code seems to follow that discipline, which seems unusual to me. I think of stderr as reserved for genuine error output.

I would suggest using stdout for any output other than error messages.